internal/app/context_table.go

```
@@ -51,6 +49,13 @@ func newContextTable() table.Model {
 	return t
 }
 
+func contextTableSelectedStyle() lipgloss.Style {
+	return selectedStyle.Copy().
```

golangci-lint (2.12.2)

[error] SA1019: selectedStyle.Copy is deprecated: to copy just use assignment (i.e. a := b). All methods also return a new style. 

(staticcheck)